source/styleEditor.js

```diff
@@ -1,3 +1,11 @@
-$(document).ready(function(){
-  // your code goes here.
-});
+
+
+$("form").submit(styleCss);
```

Comment:
I like that you passed a function to the event handler. Great for code organization